### DISCUSSION TITLE
Is it possible to turn off progress bar?

### DISCUSSION THREAD

**Discussion by qqlearn123** [duckdb/duckdb#7990] [Q&A]
I noticed that when I convert the query results to dataframe, there is a progress bar showing up. Do we have control over it to make show or not show?

**Comment by carlopi** [comment]
`PRAGMA disable_progress_bar;` should probably do it

**Reply by qqlearn123** [reply]
> `PRAGMA disable_progress_bar;` should probably do it

Thanks. How about the equivalent in Python?

**Reply by carlopi** [reply]
I guess something along the lines of:
```python
import duckdb

duckdb.execute('PRAGMA disable_progress_bar;')
```
